test/fixtures/test-runner/NODE_TEST_WORKER_ID/1.test.js

```diff
@@ -0,0 +1,5 @@
+const test = require('node:test');
```

Comment:
It may have been missed among the other changes, but please rename this directory from `NODE_TEST_WORKER_ID` to something like `worker-id`.